.travis.yml

```diff
@@ -0,0 +1,29 @@
+language: ruby
+cache: 
+  - bundler
+  - npm
+jobs:
+  include:
+    - language: ruby
```

Comment:
can we give these jobs some descriptive names?  

```suggestion
    - name: "Test with Ruby 2.5"
    - language: ruby
```


Right now I see this:

<img width="362" alt="Screen Shot 2020-05-27 at 4 29 43 PM" src="https://user-images.githubusercontent.com/41651655/83069090-5486fe80-a037-11ea-9a9d-fa0a5a1dee85.png">

But it would be nice to have something like:

![Screen Shot 2020-05-27 at 4 30 34 PM](https://user-images.githubusercontent.com/41651655/83069171-75e7ea80-a037-11ea-9cfd-3cbb787f4512.png)

Reply:
Sure